src/ui_utils.c

```diff
@@ -2566,6 +2582,10 @@ void ui_init(void)
 	ui_init_toolbar_widgets();
 	init_document_widgets();
 	create_config_files_menu();
+
+	// after UI is initialized, apply user's custom CSS (if it exists)
+	// to override other CSS styles
+	init_user_style();
```

Comment:
I really dislike the way items are added to that menu, a bunch of scattered calls mixed in to all kinds of different modules, each calling that function in a different way. It would be nice to move this to Glade or at least have a single location that populates the menu (which could itself be in glade at least).